Jellyfin.Api/Auth/FirstTimeSetupPolicy/FirstTimeSetupHandler.cs

```diff
@@ -38,7 +38,15 @@ protected override Task HandleRequirementAsync(AuthorizationHandlerContext conte
                 return Task.CompletedTask;
             }
 
-            if (requirement.RequireAdmin && !context.User.IsInRole(UserRoles.Administrator))
+            var contextUser = context.User;
+            if (requirement.RequireAdmin && !contextUser.IsInRole(UserRoles.Administrator))
+            {
+                context.Fail();
+                return Task.CompletedTask;
+            }
+
+            var userId = contextUser.GetUserId();
+            if (userId.Equals(default))
```

Comment:
This makes it so api keys can't use the endpoint- is that what is desired?

Reply:
API keys are handled in the `DefaultAuthorizationHandler`. The check over there only gets skipped if the request does not incude an API key and the default userID is used.